source/objects/RenderTarget.cpp

```diff
@@ -247,6 +247,17 @@ void RenderTarget::GenerateMIPs()
 	}
 }
 
+IRenderTarget* RenderTarget::Clone() {
+	RenderTarget* rt = new RenderTarget(GetWidth(), GetHeight(), GetFormat(), mMips);
```

Comment:
The reason I said to use a copy constructor is because this initialises the entire image first, whereas a copy constructor means you can just copy the member variables across